src/logic/Window.swift

```diff
@@ -184,16 +184,29 @@ class Window {
                 guard let self else { return }
                 var psn = ProcessSerialNumber()
                 GetProcessForPID(self.application.pid, &psn)
+
+                self.deactivateWindow(&psn)
+
                 _SLPSSetFrontProcessWithOptions(&psn, self.cgWindowId!, SLPSMode.userGenerated.rawValue)
                 self.makeKeyWindow(&psn)
-                self.axUiElement!.focusWindow()
+
+                usleep(10000)
+                self.axUiElement?.focusWindow()
+
                 DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(50)) {
                     Windows.previewFocusedWindowIfNeeded()
                 }
             }
         }
     }
 
+    func unfocus() {
+        Logger.info("Unfocusing \(application.bundleURL.map { "\($0)" } ?? "nil") \(cgWindowId.map { "\($0)" } ?? "nil")")
+        var psn = ProcessSerialNumber()
+        GetProcessForPID(self.application.pid, &psn)
+        self.deactivateWindow(&psn)
```

Comment:
This is what seems to be causing the window ordering to break. I'm guessing it causes a focus event to be sent, but I'm not sure how to prevent AltTab from handling it.